modules/nf-core/sentieon/bwaindex/main.nf

```diff
@@ -0,0 +1,69 @@
+process SENTIEON_BWAINDEX {
+    tag "$fasta"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Sentieon modules does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container 'nfcore/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path(bwa), emit: index
+    path "versions.yml"       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    if [ \${SENTIEON_LICENSE_BASE64:-"unset"} != "unset" ]; then
+        echo "Initializing SENTIEON_LICENSE env variable"
+        if [ "\${#SENTIEON_LICENSE_BASE64}" -lt "1500" ]; then # Sentieon License server
+            export SENTIEON_LICENSE=\$(echo -e "\$SENTIEON_LICENSE_BASE64" | base64 -d)
+        else  # Localhost license file
+            export SENTIEON_LICENSE=\$(mktemp)
+            echo -e "\$LICENSE_ENCODED" | base64 -d > \$SENTIEON_LICENSE
+        fi
+    fi
+
+    mkdir bwa
+
+    sentieon \\
+        bwa index \\
+        $args \\
+        -p bwa/${fasta.baseName} \\
+        $fasta
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        sentieon: \$(echo \$(sentieon driver --version 2>&1) | sed -e "s/sentieon-genomics-//g")
+        bwa: \$(echo \$(sentieon bwa 2>&1) | sed 's/^.*Version: //; s/Contact:.*\$//')
+    END_VERSIONS
+    """
+
+    stub:
+    """
+    mkdir bwa
+
+    touch bwa/genome.amb
+    touch bwa/genome.ann
+    touch bwa/genome.bwt
+    touch bwa/genome.pac
+    touch bwa/genome.sa
```

Comment:
```suggestion
for ext in amb ann bwt pac sa; do
    touch bwa/genome.$ext
done
```
Just avoiding repetition/ saving a couple of lines

Reply:
Love it, but don't think it's that important, it's just stub, and we're just saving 2 lines